client/src/components/NavBar.js

```diff
@@ -10,73 +10,87 @@ class NavBar extends Component {
         ? "home"
         : window.location.pathname.substr(1)
   };
+  // handlers
   handleItemClick = (e, { name }) =>
     this.setState({ activeItem: name, visible: false });
 
-  logout = event => {
-    event.preventDefault();
-    localStorage.removeItem("token");
-    document.location.reload();
-  };
-  handleHideClick = () => this.setState({ visible: false });
   handleShowClick = () =>
     this.setState(prevState => {
       const visibleState = prevState.visible;
       return {
         visible: !visibleState
       };
     });
-  handleSidebarHide = () => this.setState({ visible: false });
+
+  //logOut function
+  logout = event => {
+    event.preventDefault();
+    localStorage.removeItem("token");
+    document.location.reload();
+  };
 
   render() {
     const { visible, activeItem } = this.state;
-    const { handleSidebarHide, handleItemClick, logout } = this;
+    const { handleItemClick, logout } = this;
     return (
       <React.Fragment>
-        <Menu inverted>
-          <Menu.Item>
-            <Button disabled={visible} onClick={this.handleShowClick}>
-              <Icon name="list layout"></Icon>
-            </Button>
+        <Menu size="mini" flued inverted>
+          <Menu.Item onClick={this.handleShowClick}>
```

Comment:
can we be consistent please around how we reference functions in the render method, and add this to the destructuring statement on line 34?